tests/v2/validation/upgrade/config.go

```diff
@@ -16,6 +16,7 @@ const (
 	latestKey                  = "latest"       // latestKey is a string to determine automatically version pooling to the latest possible
 	RancherPrivileged    PSACT = "rancher-privileged"
 	RancherRestricted    PSACT = "rancher-restricted"
+	RancherBaseline      PSACT = "rancher-baseline"
```

Comment:
this Const in a few places. Can we remove it from here and import from provisioninginput?

Reply:
Agreed, we should already have `RancherPrivileged` and `RancherRestricted` there, too. I think we can import all of them from `provisioninginput`

Reply:
agreed